Sources/ArcGISToolkit/Components/Offline/PreplannedMetadataView.swift

```diff
@@ -0,0 +1,125 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct PreplannedMetadataView: View {
+    /// The view model for the preplanned map.
+    @ObservedObject var model: PreplannedMapModel
+    
+    /// The action to dismiss the view.
+    @Environment(\.dismiss) private var dismiss: DismissAction
+    
+    public var body: some View {
+        Form {
```

Comment:
The top padding here between the title and form looks larger than other SwiftUI sheets. Can this padding be reduced or is it SwiftUI's automatic layout?

<img src="https://github.com/user-attachments/assets/f82fd8c3-8731-47fd-bfbc-d852dcd053eb" width="300">

Reply:
[Here](https://stackoverflow.com/questions/57851878/how-to-eliminate-the-space-above-section-in-swiftui-form) are some hacks to eliminate the empty space, though I don't like any of them 🤔 It seems the space is reserved for the section header and even if the header isn't specified, the space is still occupied. Maybe SwiftUI will change the appearance in the future.

Reply:
Okay thanks. It's probably better to not add a workaround for this issue then.